.github/workflows/check-licenses.yml

```diff
@@ -27,18 +27,26 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Clone License Check Repo
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           repository: eclipse-velocitas/license-check
-          ref: v1
+          ref: topic/prepare_release
```

Comment:
I would prefer to go for GitHub Action referencing convention v1, v2, v3 instead of other customise ref unless there is a technical limitation on that. I think also the ref (topic/prepare_releas) may somehow lead to confusion of which release.

Reply:
That's just currently for some tryouts